### DISCUSSION TITLE
Regarding Problem 5_ Task 05 Aruco Marker MAT file

### DISCUSSION THREAD

**Discussion by AnasAlsharo** [chulminy/CIVE497-CIVE700#166] [Q&A]
Hi Dr. Yeum. 

When I try to print the mat['aruco_marker'] it gives the following output:

[[[0 0 1 0 0]
  [0 0 0 1 1]
  [0 1 0 0 1]
  [0 1 1 1 1]]

 [[1 0 1 0 1]
  [1 0 0 1 0]
  [1 1 0 0 0]
  [1 1 1 0 1]]

 [[1 0 0 1 1]
  [0 0 1 0 1]
  [0 1 0 0 0]
  [1 0 0 1 0]]

 [[1 1 0 1 1]
  [0 1 1 1 1]
  [1 0 1 1 0]
  [0 1 0 0 1]]]
Which is a 4- (4x5) binary matrices. I would expect having 5 - (4x4) matrices as we have five distinct markers each is 4x4. 

Am I getting anything wrong here? 

Thank you!

**Comment by chulminy** [comment]
I'm a matlab guy so I was also confused. It's a way of how Python stores the data. See my example:
![image](https://github.com/chulminy/CIVE497-CIVE700/assets/3227103/8e5961cb-3fd7-4651-992b-c9025549a77c)

**Reply by AnasAlsharo** [reply]
Oh! got it! 
Thank you!
